include/call_interfaces.h

```diff
@@ -167,6 +167,7 @@ struct sdp_ng_flags {
 	long long duration;
 	long long pause;
 	long long start_pos;
+	bool flush_delay_buffer;
```

Comment:
The `flags` struct is usually reserved for things set by the user / controlling agent, so I'd prefer to pass this bool down some other way if at all possible.  How about adding it to `struct chu_args`, which would make it available in the `a` argument, and then pass it as an argument to `codec_update_all_source_handlers`?

Reply:
ok, yeah that makes sense, thanks.
Have made the change but flipped the bool to be `clear_delay_buffer` as it will be false when not explicitly passed in